controller/work_item_link_blackbox_test.go

```diff
@@ -60,10 +64,10 @@ type workItemLinkSuite struct {
 // The TearDownSuite method will run after all the tests in the suite have been run
 // It tears down the database connection for all the tests in this suite.
 func (s *workItemLinkSuite) TearDownSuite() {
-	if s.DB != nil {
-		s.DB.Unscoped().Delete(&account.Identity{Username: "test user"})
-		s.DB.Close()
-	}
+	// if s.DB != nil {
```

Comment:
right, I commented out the lines to verify that the code would still work but then I forgot to remove them. It's done, now.